src/style.css

```
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+}
```

```suggestion
body {
  margin: 0;
}
```
it is redundant reset margins for all selectors.
reset just for elements where it is necessary

* selector generally just for one property: `box-sizing: border-box`